File: Restaurant/Java/Drink.java
package restaurant;

class Drink extends Item {
	
	Drink(){	
	}
	
	Drink(String name, double price){
		this.name = name;
		this.price = price;
	}
	
	String getName() {
		return this.name;
	}
	
	double getPrice() {
		return this.price;
	}
}

File: Restaurant/C++/Drink.cpp
#include "Drink.hpp"
#include <string>

Drink::Drink()
{}

Drink::~Drink()
{}

Drink::Drink(std::string name, double price)
{
	this->name = name;
	this->price = price;
}

std::string Drink::getName()
{
	return this->name;
}

double Drink::getPrice()
{
	return this->price;
}